content/03.results.md

```diff
@@ -1,8 +1,8 @@
 ## Results
 
-### Crowd-sourced Somatic Analysis to create an Open Pediatric Brain Tumor Atlas
+### Crowd-sourced Somatic Analyses to Create an Open Pediatric Brain Tumor Atlas
 
-We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines from 943 patients from the Pediatric Brain Tumor Atlas (PBTA) [@doi:10.1093/neuonc/noz192], consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258; @doi:10.1158/1078-0432.CCR-22-0803] of the Pacific Pediatric Neuro-Oncology Consortium (PNOC) (**Figure {@fig:Fig1}A**).
+We previously performed whole genome sequencing (WGS), whole exome sequencing (WXS), and RNA sequencing (RNA-Seq) on matched tumor and normal tissues as well as selected cell lines [@doi:10.1093/neuonc/noz192] from 943 patients from the Pediatric Brain Tumor Atlas (PBTA), consisting of samples from the [Children’s Brain Tumor Network (CBTN)](https://CBTN.org) and the PNOC003 DMG clinical trial [@doi:10.1002/ijc.32258; @doi:10.1158/1078-0432.CCR-22-0803] of the Pacific Pediatric Neuro-Oncology Consortium (PNOC) (**Figure {@fig:Fig1}A**).
```

Comment:
I'd remove the CBTN link. Or, for consistency, also add a PNOC link.

Reply:
I added the PNOC link. I didn't have that since I had the papers for PNOC and CBTN did not have any papers yet.